Livecheckables/libtorch.rb

```diff
@@ -0,0 +1,4 @@
+class Libtorch
+  livecheck :url   => "https://github.com/pytorch/pytorch/releases",
+            :regex => %r{latest.*?href="/pytorch/pytorch/tree/v?(\d+(?:\.\d+)+)}m
```

Comment:
```suggestion
  livecheck :url   => "https://github.com/pytorch/pytorch/releases/latest",
            :regex => %r{href=.+/tag/v?(\d+(?:\.\d+)+)}
```

As in the `mupen64plus` PR, we can use the "latest" release page here and also use the same general regex that we used there (finding tag URLs) to get the version.